Bottom: 84d0745a024d7009a9b60d8ea6751c6abf1e3262
Top:    f9af2618a5c79a2efa3100258166734f07f652f4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 02:36:59 +0000

Refresh of tag-circular-contours-so-they--I

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 1661526..1c7ead9 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -183,14 +183,6 @@ void common_fill_pcb_polygon (hidGC gc, PolygonType *poly,
 static int thindraw_hole_cb (PLINE *pl, void *user_data)
 {
   hidGC gc = user_data;
-
-  /* If the contour is round, use an arc drawing routine. */
-  if (pl->is_round) {
-    gui->set_line_width (gc, 0);
-    gui->draw_arc (gc, pl->cx, pl->cy, pl->radius, pl->radius, 0, 360);
-    return 0;
-  }
-
   thindraw_contour (gc, pl);
   return 0;
 }
